docs/reference/platforms.rst

```diff
@@ -13,22 +13,22 @@ OS X
 
 .. image:: /reference/screenshots/cocoa.png
 
-The backend for OS X is named `toga-cocoa`_. It supports OS X 10.7 (Lion)
+The backend for OS X is named `toga_cocoa`_. It supports OS X 10.7 (Lion)
 and later. It is installed automatically on OS X machines (machines that
 report ``sys.platform == 'darwin'``), or can be manually installed by invoking::
 
     $ pip install toga[cocoa]
 
 The OS X backend has seen the most development to date.
 
-.. _toga-cocoa: http://github.com/pybee/toga-cocoa
+.. _toga-cocoa: https://github.com/pybee/toga/tree/master/src/cocoa
```

Comment:
Should this be 
`.. _toga_cocoa:`

Reply:
Yes, thanks. I need to make several more corrections.